src/components/signon/types.ts

```diff
@@ -0,0 +1,20 @@
+export type SignOnStackParamList = {
+  SignOn: undefined
```

Comment:
what is this type

Reply:
This defines the types of the params for each screen in the navigator. `undefined` means there are no params for that screen